Makefile

```diff
@@ -18,12 +18,18 @@ EDGE_VERSION ?= 0.15.0
 REGISTRY_DIR=~/.terraform.d/plugins/registry.terraform.io/juju/juju/${EDGE_VERSION}/${GOOS}_${GOARCH}
 
 .PHONY: install
-install: simplify docs go-install
+install: simplify docs schema go-install
 ## install: Build terraform-provider-juju and copy to ~/.terraform.d using EDGEVERSION
 	@echo "Copied to ~/.terraform.d/plugins/registry.terraform.io/juju/juju/${EDGE_VERSION}/${GOOS}_${GOARCH}"
 	@mkdir -p ${REGISTRY_DIR}
 	@cp ${GOPATH}/bin/terraform-provider-juju ${REGISTRY_DIR}/terraform-provider-juju_v${EDGE_VERSION}
 
+.PHONY: schema
+schema:
+## schema: add provider's schema to terraform provider folder to enable language server features (ex. autocomplete).
+	@terraform providers schema -json > ${REGISTRY_DIR}/schema.json
```

Comment:
issue: trying to put something in a directory which hasn't been created yet. It's created in the `install` make target.

```
/bin/sh: 1: cannot create /home/heather/.terraform.d/plugins/registry.terraform.io/juju/juju/0.15.0/linux_amd64/schema.json: Directory nonexistent
```